src/Microsoft.AspNetCore.Routing/RequestDelegateRouteBuilderExtensions.cs

```diff
@@ -75,6 +76,22 @@ public static IRouteBuilder MapDelete(this IRouteBuilder builder, string templat
         }
 
         /// <summary>
+        /// Adds a route to the <see cref="IRouteBuilder"/> that only matches HTTP DELETE requests for the given
+        /// <paramref name="template"/>, and <paramref name="handler"/>.
+        /// </summary>
+        /// <param name="builder">The <see cref="IRouteBuilder"/>.</param>
+        /// <param name="template">The route template.</param>
+        /// <param name="handler">The route handler.</param>
+        /// <returns>A reference to the <paramref name="builder"/> after this operation has completed.</returns>
+        public static IRouteBuilder MapDelete(
+            this IRouteBuilder builder,
+            string template,
+            Func<HttpRequest, HttpResponse, RouteData, Task> handler)
```

Comment:
Good question. You certainly don't want the DataTokens (there aren't any) or the Routers (super-not-interesting here). So perhaps RVD is all that's even useful anyway?